conans/client/rest/conan_requester.py

```diff
@@ -55,6 +56,9 @@ def _add_kwargs(self, url, kwargs):
                 kwargs["proxies"] = self.proxies
         if self._timeout_seconds:
             kwargs["timeout"] = self._timeout_seconds
+        if not kwargs.get("headers"):
+            kwargs["headers"]  = {}
+        kwargs["headers"]["User-Agent"] = "Conan/%s (Python %s)" % (client_version,python_version.split()[0])
```

Comment:
Better use `platform.python_version()` it automatically extracts the python version